UIKitDZ/UIKitDZ/Modules/AddIngredientsModule/Model/AddIngredientsModel.swift

```diff
@@ -0,0 +1,15 @@
+// AddIngredientsModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Структура для хранения информации овыбранных ингредиентов
+class AddIngredientsModel {
+    static var statesDictionaryMap: [String: Bool] = [
```

Comment:
```suggestion
    static var statesMap: [String: Bool] = [
```
Dictionary стоп слово